CMakeLists.txt

```diff
@@ -818,6 +841,14 @@ endif()
 # --------------------------------------------------------------------------- #
 # Configure make install/uninstall and packages
 # --------------------------------------------------------------------------- #
+# Install umfd target
+if(WINDOWS AND UMF_USE_DEBUG_POSTFIX)
+    install(FILES ${CMAKE_BINARY_DIR}/bin/Debug/umfd.dll
+            DESTINATION ${CMAKE_INSTALL_BINDIR})
+    install(FILES ${CMAKE_BINARY_DIR}/lib/Debug/umfd.lib
+            DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
And so do these. Also unable to install only these because there's no `COMPONENT` to name.